.github/workflows/ci.yaml

```diff
@@ -111,9 +112,10 @@ jobs:
         run: npm run compile
 
       - name: Run ${{ matrix.npm-target }}
-        uses: GabrielBB/xvfb-action@v1.0
-        with:
-          run: npm run ${{ matrix.npm-target }}
+        # uses: GabrielBB/xvfb-action@v1.0
+        # with:
+        run: |
+          xvfb-run --server-args="-screen 0, 1920x1080x24" npm run ${{ matrix.npm-target }}
```

Comment:
The action already solves this, no need for a manual command.

Reply:
`xvfb-action` is unmaintained and buggy and I remember finding at least two people online that stopped using it and starting to use `xvfb-run` directly, especially because they reported that the default screen was too small and caused errors running tests. 

BTW, that PR was a draft and i am not working on it. I rely on @priyamsahoo to do it, that is why I marked it as draft.  i will likely close it soon.

Reply:
In that case, I'd rather fork the action. Its UX is a bit more pleasant + reusable.